src/common/AdaptiveTree.h

```diff
@@ -429,7 +429,8 @@ class ATTR_DLL_LOCAL AdaptiveTree
 
     std::vector<AdaptationSet*> adaptationSets_;
     std::string base_url_, id_;
-    uint32_t timescale_ = 1000, startNumber_ = 1, sequence_ = 0;
+    uint32_t timescale_ = 1000, sequence_ = 0;
```

Comment:
since you changing existing code variables must be placed on each line, not in-line